src/schedulion-react-fe/src/components/Statistics.js

```diff
@@ -1,6 +1,4 @@
 import React from 'react';
-import Button from 'react-bootstrap/Button'
-import { BrowserRouter, Switch, Route, Link } from 'react-router-dom';
 import './Statistics.css';
 // import { }from 'react-bootstrap-icons';
```

Comment:
Not necessarily your code, but I'm unsure if this is necessary. Commented code should be deleted before pushing.